ocfweb/login/urls.py

```diff
@@ -1,12 +1,15 @@
-from django.conf.urls import url
+from django.conf.urls import include, url
 
+from ocfweb.account.urls import urlpatterns as account
 from ocfweb.login.calnet import login as calnet_login
 from ocfweb.login.calnet import logout as calnet_logout
 from ocfweb.login.ocf import login
 from ocfweb.login.ocf import logout
 
 
 urlpatterns = [
+    url(r'^account/', include(account)),
+
```

Comment:
Why do all these changes need to be made? I believe the link will just work without all of this.

Reply:
Oh, you are right, it is already included in the main urlpatterns.